pkg/cloud/services/interfaces.go

```diff
@@ -31,7 +31,9 @@ type Getter interface {
 
 // SDKSessionGetter has a single method that returns an AWS session.
 type SDKSessionGetter interface {
-	Session(*providerv1.AWSClusterProviderSpec) *session.Session
+	//FIXME
+	//Session(*providerv1.AWSClusterProviderSpec) *session.Session
```

Comment:
Is the SDKSessionGetter interface used anywhere outside of the mocks?

Reply:
It doesn't appear to be used anywhere.